manageiq-operator/pkg/helpers/miq-components/httpd.go

```diff
@@ -72,7 +83,7 @@ func NewHttpdConfigMap(cr *miqv1alpha1.ManageIQ) *corev1.ConfigMap {
 			Labels:    labels,
 		},
 		Data: data,
-	}
+	}, nil
```

Comment:
Can you just put the config map into a local so this is a bit easier to read?

Reply:
👍  like master, will do.